src/petunia.js

```diff
@@ -0,0 +1,23 @@
+import 'babel-polyfill';
+import { Crypto } from './crypto.js';
+
+export class Petunia {
```

Comment:
Why Petunia then Crypto, it feels redundant.

Reply:
Petunia class is an adapter to expose public methods.
In this first version I agree it looks redundant because it's basically exposing the same that we have in the crypto class. But in the future we may have more classes with different responsibilities, like aws helpers, etc.